### PR TITLE
[release 2.3] update org references to stolostron

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

Work in progress

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 # Build the manager binary
-FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS builder
+FROM registry.ci.openshift.org/stolostron/builder:go1.16-linux AS builder
```
Comment (jlpadilla) [inline Dockerfile:4]: Should we update to go1.17 too?

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml [hunk 2]
```diff
@@ -13,4 +13,4 @@ spec:
   pullpolicy: Always
   pullsecret: multiclusterhub-operator-pull-secret
   searchimageoverrides:
-    redisgraph_tls: quay.io/open-cluster-management/redisgraph-tls:2.2.0-PR40-82f419595a22d6f15538ae2379b3f286e0273539
+    redisgraph_tls: quay.io/stolostron/redisgraph-tls:2.2.0-PR40-82f419595a22d6f15538ae2379b3f286e0273539
```
Comment (jlpadilla) [inline config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml:16]: This image tag doesn't exist in the new org.  Just checking that we don't use it.  Maybe change it to a dummy value if it's only a sample.

File: vbh/build-harness-extensions/README.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ In order to use the build harness and extensions, make yourself a [token](https:
 GITHUB_USER := $(shell echo $(GITHUB_USER) | sed 's/@/%40/g')
 GITHUB_TOKEN ?=
 
--include $(shell [ -f ".build-harness-bootstrap" ] || curl -sSL -o .build-harness-bootstrap -H "Authorization: token $(GITHUB_TOKEN)" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.com/open-cluster-management/build-harness-extensions/master/templates/Makefile.build-harness-bootstrap"; echo .build-harness-bootstrap)
+-include $(shell [ -f ".build-harness-bootstrap" ] || curl -sSL -o .build-harness-bootstrap -H "Authorization: token $(GITHUB_TOKEN)" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.com/stolostron/build-harness-extensions/master/templates/Makefile.build-harness-bootstrap"; echo .build-harness-bootstrap)
```
Comment (jlpadilla) [inline vbh/build-harness-extensions/README.md:14]: I believe `master` was renamed to `main`.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Approving, but please review comments before merging.

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml (hunk 2 above)
Comment (jlpadilla) [reply config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml:16]: I believe this is why Travis failed.

File: Dockerfile (hunk 1 above)
Comment (SherinV) [reply Dockerfile:4]: I have those changes in https://github.com/stolostron/search-operator/pull/94

Comment (jlpadilla) [reply Dockerfile:4]: Thanks, approved #94.

File: vbh/build-harness-extensions/README.md (hunk 3 above)
Comment (SherinV) [reply vbh/build-harness-extensions/README.md]: Updated. Thanks.

File: config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml (hunk 2 above)
Comment (SherinV) [reply config/samples/search.open-cluster-management.io_v1alpha1_searchoperator.yaml]: Updated to latest tag in stolostron.

**Pull request merged**